lib/enums/exitCodes.ts

```diff
@@ -1,13 +1,10 @@
-// @ts-nocheck
 /*
  * 0: Successful run
  * 1: Config problem or internal error
  * 2: Warnings or validation issues
  */
-const EXIT_CODES = {
+export const EXIT_CODES = {
   SUCCESS: 0,
   ERROR: 1,
   WARNING: 2,
-};
-
-module.exports = { EXIT_CODES };
+} as const;
```

Comment:
Thought about maybe moving this into `constants` and deleting this file, but its used in so many different places and didn't want to make this PR confusing